### PR TITLE
Fix MEM consuming huge amounts of disk space due to texture cache being cleared far too often

### PR DESCRIPTION
**Pull request by Mgamerz**
In the mip map replace code, there is a check for the TFC name on the export matching the TFC textures are being installed to. If these don't match, the existing cached texture data (where the texture was last installed to a TFC, offsets) is cleared, which causes it to rewrite textures to the TFC. I think this is leftover from OT when TFCs were rebuilt rather than appended to. This check doesn't seem to really do anything because in later code the name and guid of the tfc housing the new data is always written to the export.

This comments out the aforementioned check (kept in case it's actually needed for something) but I had some people test it and they reported:

 - Installation is much faster on big mods like LEUITM and Primatives for LE1
 - TFC data is FAR smaller - for LEUITM its a couple hundred MB vs tens of GBs
 - Textures all appear to work fine

### CODE REVIEW

File: MassEffectModder/MassEffectModder/MipMaps/MipMapsReplace.cpp [hunk 1]
```diff
@@ -677,10 +677,14 @@ QString MipMaps::replaceTextures(QList<MapPackagesToMod> &map, QList<TextureMapE
...
-                if (mod.arcTfcDLC && mod.arcTfcName != archive)
-                {
-                    mod.arcTexture.clear();
-                }
+
+                // Commented out on June 23, 2023
+                // Appears to be leftover from when TFCs were rebuilt in OT MEM
+                // Leaving here in case it is needed if an issue is found
+            	// if (mod.arcTfcDLC && mod.arcTfcName != archive)
+				// {
```
Comment (Mgamerz) [inline MassEffectModder/MassEffectModder/MipMaps/MipMapsReplace.cpp:685]: visual studio decided to indent these -_-

**Comment by aquadran** [comment]
I'll take look this

**Comment by Yamakuzure** [comment]
Just to post some experience with the consequences of what this "leftover" means:

My LE1 folder got from 15.6 GB up to 23.8 GB after installing all content mods.
And then it blasted up to 127GB (No Joke!) after texture modding with ALOT, ISL, Primitives and LEUITM (*).
This was a **one shot** texture install. Currently I can not mod LE2 or LE3 if I mod LE1.

One-Shot-Installs are important, because I then installed the BroShep and LiaraLOTSB textures over, thus after Primitives, to test something. My LE1 folder got pumped up to 146GB from this. So I installed two mem files with under 300MB, and got 19 GB out of it.

I do understand that mem files are compressed and that they are supposed to grow the ME1 folder by a lot more than what their mem size is, also by textures possibly being installed in more than one place. But 300MB => 19000MB is ... erm... _a bit_ much, maybe?

(*) There are a lot of overlapping textures with this combination, so that may add to the growth.

**Comment by mirh** [comment]
Actually, putting even aside of this obvious bug, if MEM used some dxt compressor with rate distortion optimizations you could even cut their final package size by half.

**Comment by aquadran** [comment]
code applied into  ef179eecf1e61f8bf431328d9722ae43b97360df

**Pull request closed without merging**